lib/engines/content_object_store/features/support/stubs.rb

```diff
@@ -1,12 +1,3 @@
 Before do
-  stub_dependent_content(content_id: anything, total: 0, results: [])
-end
-
-def stub_dependent_content(results:, content_id: anything, total: 0)
-  stub_request(:get, %r{\A#{Plek.find('publishing-api')}/v2/content/[0-9a-fA-F-]{36}/embedded})
-    .to_return(body: {
-      "content_id" => content_id,
-      "total" => total,
-      "results" => results,
-    }.to_json)
+  stub_publishing_api_has_embedded_content(content_id: anything, total: 0, results: [])
```

Comment:
so  `stub_publishing_api_has_embedded_content` is defined in the adapters repo?

Reply:
Yeah that's right. https://github.com/alphagov/gds-api-adapters/pull/1283/commits/b206bf80c7a53f1148a6a9cb77e54ee3a101bd9c. I think it's because that way any consumer has easy access to the same stubs, rather than having to redeclare them.